src/styles/styles.js

```diff
@@ -1230,6 +1230,11 @@ const styles = {
     appContent: {
         backgroundColor: themeColors.appBG,
         overflow: 'hidden',
+
+        // it's needed to prevent incorrect triple click text selection in chromium and webkit based browsers
```

Comment:
The comment needs to be changed, though I am not sure how we should write it.

AFAIK the issue occurs when the property `user-select` is set as 'none' at the parent and `user-select: 'text'` at the child (as displayed in the [codepen example](https://codepen.io/wildanmuhlis2/pen/KKeWGQr?editors=1000))

In our case the drawer is adding `user-select: 'none'` which we can't change so we are passing  `user-select: 'auto'` at [ScreenWrapper](https://github.com/Expensify/App/blob/c19e245c77bd9fadc72a49560f27ff3036a4763b/src/pages/home/ReportScreen.js#L231) instead 

@chiragsalian, please help with the comment.

Reply:
@thesahindia actually to reproduce it, we should also have an input field and text after the input field at the end of the container.

What if we change the comment like this one:

```
// it's needed to prevent incorrect triple-click text selection.
// Starting version 6.3.2 @react-navigation/drawer add user-select: none; to its container.
// The property addition create an issue when dealing with this test case
// ----------------------------------
// <div style="user-select: none;">
//    <div style="user-select: text;">lorem</div>
//    <div style="user-select: text;">please triple click me</div>
//    <input type="text" value="I shouldn't be selected" />
//  </div>
//  <div>ipsum</div>
// -----------------------------------
//  see: https://codepen.io/wildanmuhlis2/pen/KKeWGQr
// add user-select-auto to the inner container will normalize/restore the child DOM and resolve the issue
// Related issue: https://github.com/Expensify/App/issues/12028
```

Reply:
I like the comment with more detail as you suggested @wildan-m.

I would change the `add` in this part,
> Starting version 6.3.2 @react-navigation/drawer add user-select: none; to its container.

to `adds`, since its something happening by default and we don't need to add it.

I would prefer it slightly rephrased to state problem first and then explain solution, so I'm thinking something like this,
```
// Starting version 6.3.2 @react-navigation/drawer adds "user-select: none;" to its container.
// We add user-select-auto to the inner component to prevent incorrect triple-click text selection.
// For further explanation see - https://github.com/Expensify/App/pull/12730/files#r1022883823
```

What do you think @thesahindia @wildan-m?

Reply:
Looks good to me!

Reply:
Looks good, just updated the comment